tests/query/processors/test_org_rate_limiter.py

```diff
@@ -102,3 +102,33 @@ def test_org_rate_limit_processor_overridden(
     assert rate_limiter.bucket == str(org_id)
     assert rate_limiter.per_second_limit == 5
     assert rate_limiter.concurrent_limit == 10
+
+
+def test_namespaced_rate_limit() -> None:
```

Comment:
I don't really understand why this test passes. You're writing a config without a namespace, reading it with a namespace and somehow it works but I'm not sure why

Reply:
Because the test should copy the value to the new namespace.

1) Write a value to the old namespace (no config key)
2) Run the test, which should copy the value to the new namespace
3) Check the value was copied (rate limit config key)